dali/pipeline/operators/resize/random_resized_crop.cc

```diff
@@ -29,15 +29,20 @@ DALI_SCHEMA(RandomResizedCrop)
   .NumOutput(1)
   .AllowMultipleInputSets()
   .AddOptionalArg("random_aspect_ratio",
-      R"code(Range from which to choose random aspect ratio.)code",
+      R"code(Range from which to choose random aspect ratio (width/height).)code",
       std::vector<float>{3./4., 4./3.})
   .AddOptionalArg("random_area",
       R"code(Range from which to choose random area factor `A`.
 Before resizing, the cropped image's area will be equal to `A` * original image's area.)code",
       std::vector<float>{0.08, 1.0})
   .AddOptionalArg("interp_type",
-      R"code(Type of interpolation used.)code",
+      R"code(Type of interpolation used. Use `min_filter` and `mag_filter` to specify
+      different filtering for downscaling and upscaling.)code",
       DALI_INTERP_LINEAR)
+  .AddOptionalArg("mag_filter", "Filter used when scaling up",
```

Comment:
Is random_resized_crop using new resize kernel? Because I don't see that change inside the operator.

Reply:
Not yet. Removed.